ops/testing.py

```diff
@@ -405,16 +405,43 @@ def add_relation(self, relation_name: str, remote_app: str) -> int:
         self._emit_relation_created(relation_name, rel_id, remote_app)
         return rel_id
 
+    def remove_relation(self, relation_name: str, remote_app: str) -> None:
+        """Remove a relation.
+
+        Args:
+            relation_name: The relation on Charm that is being related to
+            remote_app: The name of the application that is being related to
+
+        Raises:
+            ValueError: if relation_name is invalid
+            RelationNotFoundError: if valid relation id for relation_name is not found
+        """
+        rel_id = self._backend._relation_ids_map[relation_name][0]
```

Comment:
@jameinel Good point. Thank you. I agree that allowing developers to remove a relation without bothering to remove each unit would be a more friendly API. I presume this behaviour is also supported by Juju, so behaviour of Harness and Juju are aligned. 

Since this same pull request also implements `remove_relation_unit` would it be acceptable instead to do
```
for unit_name in self._backend._relation_list_map[rel_id]:
    self.remove_relation_unit(rel_id, unit_name)
```

Reply:
Done. Also modified unit test.

Reply:
should remove relation not be keyed on relation_id? relations have multiplicity and developers will need to test this type of scenario where individual relations disppear.